base/loading.jl

```diff
@@ -419,8 +420,9 @@ function require(mod::Symbol)
             cachefile = compilecache(mod)
             m = _require_from_serialized(1, mod, cachefile, last)
             if isa(m, Exception)
-                warn("Compilecache failed to create a usable precompiled cache file for module $name. Got:")
+                warn("`compilecache` failed to create a usable precompiled cache file for module $name. Got:")
```

Comment:
@vtjnash, no, as far as I can tell backticks are not standard punctuation marks in English.   I can't find any style guide or reference to them being used for anything other than coding purposes.